tests/Cache/AbstractHasCacheTtlInResponseTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace PodPoint\Reviews\Tests\Cache;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Request;
+use GuzzleHttp\Psr7\Response;
+use PodPoint\Reviews\Cache\AbstractHasCacheTtlInResponse;
+use PodPoint\Reviews\Cache\CacheProvider;
+use PodPoint\Reviews\Tests\TestCase;
+use PodPoint\Reviews\TimeUnitConverter;
+use Psr\SimpleCache\CacheInterface;
+
+class AbstractHasCacheTtlInResponseTest extends TestCase
+{
+    /**
+     * Mocked request used for returning getRequest.
+     *
+     * @var Request|\Mockery\LegacyMockInterface|\Mockery\MockInterface
+     */
+    protected $mockedGuzzleRequest;
+
+    /**
+     * Mock of Http Client, used to pass to api client for making requests.
+     *
+     * @var \Mockery\Mock
+     */
+    protected $httpClient;
+
+    /**
+     * Instance of mocked cached Adapter.
+     *
+     * @var \Mockery\LegacyMockInterface|\Mockery\MockInterface|CacheInterface
+     */
+    protected $cacheAdapter;
+
+    /**
+     * Preparing test instances.
+     */
+    protected function setUp()
+    {
+        $this->cacheAdapter = \Mockery::mock(CacheInterface::class);
+        CacheProvider::setInstance($this->cacheAdapter);
+
+        $this->mockedGuzzleRequest = \Mockery::mock(Request::class);
+        $this->httpClient = \Mockery::mock(Client::class, ['http_errors' => true])->makePartial();
+    }
+
+    /**
+     * Making sure that the expires_in access token
```

Comment:
Pedantic: `Making sure that the expires_in is converted in minutes and used as ttl for cache.`